contracts/ColonyNetworkStorage.sol

```diff
@@ -73,4 +73,16 @@ contract ColonyNetworkStorage is DSAuth, DSMath {
 
   // Mapping containing the last auction start timestamp for a token address
   mapping (address => uint) recentAuctions;
+
+  // Address of the ENS registrar for joincolony.eth
+  address ens;
+  // Namehash of the root node that we administer (i.e. namehash("joincolony.eth"))
+  bytes32 rootNode;
+  // Namehash of the users node that we administer (i.e. namehash("users.joincolony.eth"))
+  bytes32 userNode;
+  // Namehash of the colony node that we administer (i.e. namehash("colony.joincolony.eth"))
+  bytes32 colonyNode;
+  // Mapping from colony address to claimed label
+  mapping (address => bytes32) colonyLabels;
```

Comment:
After talking to @itspatch , it seems like we should do this for users as well, and prevent them registering a second name.

Reply:
Sounds good.

Reply:
👍